content/blog/2016/2016-11-21-serverless-ci-hyper-sh-jenkins-plugin.adoc

```diff
@@ -0,0 +1,118 @@
+---
+layout: post
+title: "Serverless CI with the new Hyper.sh plugin for Jenkins"
+tags:
+- jenkins
+- docker
+- containers
+- serverless
+- immutable servers
+- CI
+- CD
+author: jmknyc06
+---
+
```

Comment:
Please add a guest post note (see [this post as an example](https://jenkins.io/blog/2016/11/21/gc-tuning/))